querydsl-sql-codegen/src/main/java/com/querydsl/sql/codegen/ant/AntMetaDataExporter.java

```diff
@@ -644,4 +691,30 @@ public String getSourceEncoding() {
     public void setSourceEncoding(String sourceEncoding) {
         this.sourceEncoding = sourceEncoding;
     }
+
+    public String getBeansTargetFolder() {
+        return beansTargetFolder;
+    }
+
+    public void setBeansTargetFolder(String beansTargetFolder) {
+        this.beansTargetFolder = beansTargetFolder;
+    }
+
+    public TypeMapping createTypeMapping() {
+        TypeMapping mapping = new TypeMapping();
+        typeMappings.add(mapping);
+        return mapping;
+    }
+
+    public NumericMapping createNumericMapping() {
+        NumericMapping mapping = new NumericMapping();
+        numericMappings.add(mapping);
+        return mapping;
+    }
+
```

Comment:
Maybe document these?
These were necessary for ANT's element syntax right?
Maybe you can do a shoutout on the Google group if this would also work for Gradle?